components/options_resolver.rst

```diff
@@ -953,3 +953,24 @@ method ``clearOptionsConfig()`` and call it periodically::
 
 That's it! You now have all the tools and knowledge needed to process
 options in your code.
+
+Options resolver introspector
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The `OptionsResolverIntrospector` inspect options definitions inside an `OptionsResolver` instance.
+
+method::
+
+   use Symfony\Component\OptionsResolver\Debug\OptionsResolverIntrospector;
+   use Symfony\Component\OptionsResolver\OptionsResolver;
+
+    $resolver = new OptionsResolver();
+    $resolver->setDefaults([
+        'host' => 'smtp.example.org',
+        'port' => 25,
+    ]);
+
+    $introspector = new OptionsResolverIntrospector($resolver);
+    $introspector->getDefault('host'); // Retrive the default host defined value
+
```

Comment:
versionadded missing

Reply:
Not needed because this feature was first introduced in 2017.